index.html

```diff
@@ -13,7 +13,7 @@
       class="light fixed top-1 left-1 z-50 p-2 block sm:hidden border-2 border-solid rounded">
       <span class="line"></span>
     </button>
-    <nav id="nav" class="hidden fixed flex flex-col justify-center text-xl sm:flex-row sm:justify-end h-full sm:h-auto top-0 left-0 right-0 bottom-0 sm:bottom-auto bg-gray-700 sm:bg-white w-full p-4 sm:text-right">
+    <nav id="nav" class="flex fixed hidden flex-col justify-center text-xl h-full top-0 left-0 right-0 bottom-0 bg-gray-700 p-4 sm:bg-white w-full sm:flex-row sm:justify-end sm:h-auto sm:bottom-auto sm:text-right">
```

Comment:
Try `hidden sm:flex` here and remove the two `nav` rules in the CSS.

PS: It doesn't make sense to have both `flex` and `hidden` class as they set the same property (`display`).

Reply:
+ the default flex direction is `row` so you don't need `sm:flex-row`. Some other classes can also probably be removed but I'll have to run this in a browser to confirm.